### PR TITLE
Initialize calibration values when loading from metadata files

### PR DESCRIPTION
**Pull request by ziw-liu**
Previously loading calibration from a metadata file does not initialize LC values in the `QLIPP_Calibration` object created by the `load_calibration` thread worker, which [causes an error when trying to capture background afterwards](https://github.com/mehta-lab/recOrder/issues/180). This PR addresses the error by setting the required values, although the necessity of fully initializing the `QLIPP_Calibration` object when loading from metadata files is still up to debate.

### CODE REVIEW

**Comment by ziw-liu** [comment]
I'm not currently labelling this issue as automatically closing #180 upon merging. Feel free to change it as needed.

File: recOrder/plugin/workers/calibration_workers.py [hunk 1]
```diff
@@ -1,3 +1,4 @@
+from argparse import MetavarTypeHelpFormatter
```
Comment (talonchandler) [inline recOrder/plugin/workers/calibration_workers.py:1]: Does this do anything?

**Review by talonchandler — APPROVED** [review]
Thanks @ziw-liu! 

I just tested, and everything is working as I expect. I will link #180 since this fixes that issue. 

My only question is about the new import statement. Is it necessary?

As we discussed, the data flow in the calibration metadata files is a bit wonky right now. We'll design this more carefully and clean this up for 1.0.0. 

Thanks!

File: recOrder/plugin/workers/calibration_workers.py (hunk 1 above)
Comment (ziw-liu) [reply recOrder/plugin/workers/calibration_workers.py:1]: Oops that's an unintended behavior of my editor 😅

**Pull request merged**